crates/wash-cli/src/bin/wash.rs

```diff
@@ -213,7 +220,86 @@ async fn main() {
         .with_env_filter(EnvFilter::from_default_env())
         .init();
 
-    let cli: Cli = Parser::parse();
+    let mut command = Cli::command();
+    // Load plugins if they are not disabled
+    let plugins = if std::env::var("WASH_DISABLE_PLUGINS").is_err() {
```

Comment:
nit: I suppose specifying `WASH_DISABLE_PLUGINS=false` would still disable the plugins

Reply:
It would be great to have this environment variable documented at least, just a note on how defining it at all will disable plugins addresses both comments 👍🏻